rbd/pool_nautilus_test.go

```diff
@@ -99,3 +99,30 @@ func TestPoolMetadata(t *testing.T) {
 		assert.Error(t, err)
 	})
 }
+
+func TestPoolInit(t *testing.T) {
+	conn := radosConnect(t)
+	poolName := GetUUID()
+	err := conn.MakePool(poolName)
+	assert.NoError(t, err)
+
+	ioctx, err := conn.OpenIOContext(poolName)
+	assert.NoError(t, err)
+	assert.NotNil(t, ioctx)
+
+	defer func() {
+		ioctx.Destroy()
+		conn.DeletePool(poolName)
+		conn.Shutdown()
+	}()
+
```

Comment:
Can you add a test with incorrect parameters, like `ioctx == nil`?

Reply:
Added